installation/index.md

```diff
@@ -87,6 +87,36 @@ Before you can start, two decisions have to be made:
 
 Please follow the instructions in the installation article matching your platform (see the menu to the left).
 
+### getting installation info
```

Comment:
Should be `### Getting installation info`

Reply:
Done.